specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-01-31-preview/types/applications.json

```diff
@@ -0,0 +1,365 @@
+          "readOnly": true
+        }
+      },
+      "additionalProperties": false
+    },
+    "NotificationDetails": {
+      "type": "object",
+      "properties": {
+        "message": {
+          "type": "string",
+          "description": "The notification message."
+        },
+        "urgent": {
+          "type": "boolean",
+          "description": "If true, the notification is urgent."
+        }
+      },
+      "description": "The details of a user notification.",
+      "required": [
+        "message",
+        "urgent"
+      ]
+    },
+    "platformProperties": {
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "ARM id of the platform resource on which application is deployed"
+        }
+      },
+      "description": "Platform on which application is/will be deployed",
+      "required": [
+        "resourceId"
+      ]
+    },
+    "metadata": {
+      "type": "object",
+      "properties": {
+        "containingFunction": {
+          "$ref": "#/definitions/containingFunctionProperties",
+          "description": "Containing function which includes the call to faultable function"
+        },
+        "faultableFunction": {
+          "$ref": "#/definitions/faultableFunctionProperties",
+          "description": "function in which fault is injected"
+        },
+        "source": {
+          "$ref": "#/definitions/sourceProperties",
+          "description": "defines the location of the code in the class"
+        }
+      }
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Provisioning",
+        "Updating",
+        "Deleting",
+        "Accepted"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "UserAssignedManagedIdentityProperties": {
+      "type": "object",
+      "properties": {
+        "clientId": {
+          "type": "string",
+          "description": "client id of the managed identity"
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "active directory tenant id in which managed identity is provisioned"
+        },
+        "type": {
+          "type": "string",
+          "description": "type of managed identity i.e AzureManagedIdentity"
+        }
+      },
+      "description": "User assigned managed identity assigned to the platform on which application is deployed",
+      "required": [
+        "clientId",
+        "tenantId",
+        "type"
+      ]
+    },
+    "containingFunctionProperties": {
+      "type": "object",
+      "properties": {
+        "signature": {
+          "$ref": "#/definitions/signatureProperties",
+          "description": "Method signature of the containing function"
+        }
+      },
+      "description": "",
```

Comment:
>"description": "", [](http://example.com/codeflow?start=6&length=18)

remove or add text to description

Reply:
Updated the description in the document. Thanks for reviewing and catching it.